$NetBSD: patch-Makefile.am,v 1.3 2014/11/03 12:31:01 ryoon Exp $

* cp -a is not portable, DragonFly 3.2.2 has not -a option of cp at least.
* fix statement with "continue", should be '||' not '|' (pipe).

--- Makefile.am.orig	2014-08-14 10:15:46.945832128 +0000
+++ Makefile.am
@@ -250,9 +250,9 @@ if ENABLE_PLUGIN
 endif
 	${INSTALL_DATA} $(NETX_DIR)/lib/classes.jar $(DESTDIR)$(datadir)/$(PACKAGE_NAME)/netx.jar
 	${INSTALL_DATA} $(NETX_SRCDIR)/javaws_splash.png $(DESTDIR)$(datadir)/$(PACKAGE_NAME)/javaws_splash.png
-	${INSTALL_PROGRAM} launcher.build/$(javaws) $(DESTDIR)$(bindir)
-	${INSTALL_PROGRAM} launcher.build/$(itweb_settings) $(DESTDIR)$(bindir)
-	${INSTALL_PROGRAM} launcher.build/$(policyeditor) $(DESTDIR)$(bindir)
+	${INSTALL_SCRIPT} launcher.build/$(javaws) $(DESTDIR)$(bindir)
+	${INSTALL_SCRIPT} launcher.build/$(itweb_settings) $(DESTDIR)$(bindir)
+	${INSTALL_SCRIPT} launcher.build/$(policyeditor) $(DESTDIR)$(bindir)
 
 install-data-local:
 	${mkinstalldirs} -d $(DESTDIR)$(mandir)/man1
@@ -262,15 +262,15 @@ install-data-local:
 if ENABLE_DOCS
 	${mkinstalldirs} $(DESTDIR)$(htmldir)
 	(cd ${abs_top_builddir}/docs/netx; \
-	 for files in $$(find . -type f); \
+	 for files in *; \
 	 do \
-	   ${INSTALL_DATA} -D $${files} $(DESTDIR)$(htmldir)/netx/$${files}; \
+	   ${INSTALL_DATA} $${files} $(DESTDIR)$(htmldir)/netx/; \
 	 done)
 if ENABLE_PLUGIN
 	(cd ${abs_top_builddir}/docs/plugin; \
-	 for files in $$(find . -type f); \
+	 for files in *; \
 	 do \
-	   ${INSTALL_DATA} -D $${files} $(DESTDIR)$(htmldir)/plugin/$${files}; \
+	   ${INSTALL_DATA} $${files} $(DESTDIR)$(htmldir)/plugin/; \
 	 done)
 endif
 endif
@@ -483,7 +483,7 @@ stamps/netx-html-gen.stamp:
 	touch $@
 
 stamps/netx.stamp: netx-source-files.txt stamps/bootstrap-directory.stamp stamps/netx-html-gen.stamp
-	mkdir -p $(NETX_DIR)
+	mkdir -p $(NETX_DIR)/net/sourceforge/jnlp/resources
 	$(BOOT_DIR)/bin/javac $(IT_JAVACFLAGS) \
 	    -d $(NETX_DIR) \
 	    -sourcepath $(NETX_SRCDIR) \
@@ -491,14 +491,14 @@ stamps/netx.stamp: netx-source-files.txt
 	    $(NETX_CLASSPATH_ARG) \
 	    @netx-source-files.txt
 	(cd $(NETX_RESOURCE_DIR); \
-	 for files in $$(find . -type f); \
+	 for files in *; \
 	 do \
-	   ${INSTALL_DATA} -D $${files} \
-	   $(NETX_DIR)/net/sourceforge/jnlp/resources/$${files}; \
+	   ${INSTALL_DATA} $${files} \
+	   $(NETX_DIR)/net/sourceforge/jnlp/resources/; \
 	 done)
-	cp -a $(NETX_SRCDIR)/net/sourceforge/jnlp/runtime/pac-funcs.js \
+	cp -RpP $(NETX_SRCDIR)/net/sourceforge/jnlp/runtime/pac-funcs.js \
 	  $(NETX_DIR)/net/sourceforge/jnlp/runtime
-	cp -a build.properties $(NETX_DIR)/net/sourceforge/jnlp/
+	cp -RpP build.properties $(NETX_DIR)/net/sourceforge/jnlp/
 	mkdir -p stamps
 	touch $@
 
@@ -1655,7 +1655,7 @@ stamps/bootstrap-directory.stamp: stamps
 	fi;
 	mkdir -p $(BOOT_DIR)/include && \
 	for i in $(SYSTEM_JDK_DIR)/include/*; do \
-	  test -r $$i | continue; \
+	  test -r $$i || continue; \
 	  i=`basename $$i`; \
 	  rm -f $(BOOT_DIR)/include/$$i; \
 	  ln -s $(SYSTEM_JDK_DIR)/include/$$i $(BOOT_DIR)/include/$$i; \
